### DISCUSSION TITLE
How do I add snippets for C++?

### DISCUSSION THREAD

**Discussion by rakim0** [zed-industries/zed#16125] [Feature Requests]
I tried creating `~/.config/zed/snippets/cpp.json`
```jsonc
{
  "competitive programming": {
    "prefix": "cp",
    "body": ["#include <bits/stdc++.h>", "$0"],
    "description": "basic cp snippet"
  }
}
```
But it doesn't seem to work. However the javascript example found [here](https://zed.dev/docs/snippets) works fine so I think it's either filename(tried c++.json but that doesn't work too) or I'm messing up the syntax.

**Comment by bajrangCoder** [comment]
Make sure to install snippet extension.

**Reply by rakim0** [reply]
Installing the extension and renaming the snippet filename to `c++.json` works for me. Thanks!
